test/fixtures/cookbooks/influxdb-test/recipes/config_change.rb

```diff
@@ -1,3 +1,7 @@
+# NOTE: This is not idempotent since it make InfluxDB restart on every Chef run.
```

Comment:
I see.  We probably should fix this in another issue as not to distract reviewers of the main scope which is to add in ability to install by file.